python/ray/_private/runtime_env/py_executable.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from typing import List, Optional
+
+from ray._private.runtime_env.context import RuntimeEnvContext
+from ray._private.runtime_env.plugin import RuntimeEnvPlugin
+
+default_logger = logging.getLogger(__name__)
+
+
+class PyExecutablePlugin(RuntimeEnvPlugin):
```

Comment:
In the example above and in the test cases this has an implicit dependency on the `working_dir` (which makes sense to me) -- we should document that behavior explicitly in a docstring here.

Reply:
added